test/functional/wallet_transactiontime_rescan.py

```diff
@@ -136,8 +136,34 @@ def run_test(self):
         # blocks of the past 2 hours, based on the current MTP timestamp; in order to avoid
         # importing the last address (wo3), we advance the time further and generate 10 blocks
         set_node_times(self.nodes, cur_time + ten_days + ten_days + ten_days + ten_days)
-        self.generatetoaddress(minernode, 10, m1)
 
+        # send 2 btc to our third watch-only address
+        self.log.info('Send 2 btc to user')
+        miner_wallet.sendtoaddress(wo3, 2)
+
+        # generate more blocks and check blockcount
+        self.generatetoaddress(minernode, COINBASE_MATURITY, m1)
```

Comment:
It's not necessary to generate `COINBASE_MATURITY` blocks since we're not spending those coins. This also makes the previously existing comment incorrect as that states only 10 blocks are being mined.